machine-learning/immich_ml/__main__.py

```diff
@@ -12,6 +12,11 @@
 
 module_dir = Path(__file__).parent
 
+bind_host = non_prefixed_settings.immich_host
+if ":" in bind_host and not bind_host.startswith("["):
+    bind_host = f"[{bind_host}]"
+bind_address = f"{bind_host}:{non_prefixed_settings.immich_port}"
```

Comment:
I think it'd be better to use a library to check if it's IPv6. Something like this maybe? (untested)
```suggestion
from ipaddress import ip_address
bind_host = non_prefixed_settings.immich_host
if ip_address(bind_host).version == 6:
    bind_host = f"[{bind_host}]"
bind_address = f"{bind_host}:{non_prefixed_settings.immich_port}"
```

Reply:
I would like to not add extra dependencies, but if it is a better way to do it, lgtm.

Reply:
This is part of the standard library, so no extra dependency